api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -1,13 +1,15 @@
 package packit.controllers
 
 import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.CrossOrigin
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import packit.model.Packet
 import packit.service.PacketService
 
 @RestController
+@CrossOrigin
```

Comment:
I agree, I do not want to invest in creating a bean to force crossOrigin as this would change depending on final configuration for deployment in dev and prod. For example, we could overcome the issue with some settings in docker-compose.

Reply:
I was thinking we could just set the allowed origin to localhost, and then it wouldn't apply when deployed on the server so in practice we'd be forbidding cross origin access for deployed instances. Not sure how you'd do it through docker compose, would it pass through the setting in a config file?